src/pages/RandomDrinkPage.js

```diff
@@ -0,0 +1,224 @@
+          </IngredientMeasureWrapper>
+          <IngredientMeasureWrapper>
+            <dt>
+              <strong>{randomDrink.strIngredient12}</strong>
+            </dt>
+            <dd>{randomDrink.strMeasure12}</dd>
+          </IngredientMeasureWrapper>
+          <IngredientMeasureWrapper>
+            <dt>
+              <strong>{randomDrink.strIngredient13}</strong>
+            </dt>
+            <dd>{randomDrink.strMeasure13}</dd>
+          </IngredientMeasureWrapper>
+          <IngredientMeasureWrapper>
+            <dt>
+              <strong>{randomDrink.strIngredient14}</strong>
+            </dt>
+            <dd>{randomDrink.strMeasure14}</dd>
+          </IngredientMeasureWrapper>
+          <IngredientMeasureWrapper>
+            <dt>
+              <strong>{randomDrink.strIngredient15}</strong>
+            </dt>
+            <dd>{randomDrink.strMeasure15}</dd>
+          </IngredientMeasureWrapper>
+        </IngredientsList>
+      </IngredientsContainer>
+      <InstructionsContainer>
+        <SubHeading>Instructions:</SubHeading>
+        <DrinkInstruction data-testid="instructions">
+          {randomDrink.strInstructions}
+        </DrinkInstruction>
+      </InstructionsContainer>
+    </DrinkContainer>
+  );
+
+  async function getRandomDrink() {
+    try {
+      const response = await fetch(
+        'https://www.thecocktaildb.com/api/json/v1/1/random.php'
+      );
+      const data = await response.json();
+      setRandomDrink(data.drinks[0]);
+    } catch (error) {
+      console.error('ERROR:', error);
+    }
+  }
+}
+
+const DrinkContainer = styled.div`
+  display: grid;
+  gap: 25px;
+  place-items: center;
+  padding-top: 20px;
+  position: relative;
+`;
+
+const RandomDrinkButton = styled.button`
+  border-radius: 5px;
+  height: 40px;
+  font-size: 1.1rem;
+  font-family: sans-serif;
+  color: #fff;
+  background-color: #0392b0;
+  border: 2px solid #0392b0;
+  padding: 0 20px;
+
+  &:focus,
+  &:hover {
+    outline: none;
+    border-color: #f28d35;
+  }
+`;
+
+const DrinkName = styled.h2`
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  color: #f28d35;
+  max-width: 200px;
+  text-align: center;
+`;
+
+const ShortFactsList = styled.ul`
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 5px;
+
+  li {
+    font-size: 0.9rem;
+    border-radius: 20px;
+    padding: 5px 8px;
+    background-color: #f28d35;
+    color: #fff;
+  }
+`;
+
+const DrinkImage = styled.img`
+  border-radius: 5px;
+`;
+
+const IngredientsContainer = styled.div`
+  display: grid;
+  gap: 10px;
+`;
+
+const IngredientMeasureWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  gap: 60px;
+`;
+
+const SubHeading = styled.p`
+  font-weight: bold;
+  color: #f28d35;
+`;
+
+const IngredientsList = styled.dl`
+  font-size: 0.9rem;
+`;
+
+const InstructionsContainer = styled.div`
+  display: grid;
+  gap: 10px;
+  margin: 5px 10px 20px;
+`;
```

Comment:
If you use these components in other pages as well, you could outsource them into another file and reuse them in all pages. By that, you dont have to rewrite the same component over and over if you want to change the styling. Or you try to generalize your drink page into one component.

Reply:
Yes that is good point and I have planned a single user story for refactoring and styling the next days :)